src/style.css

```
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  background-color: #fff;
+}
+
+.main-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo img {
+  height: 40px;
+  margin-left: 50px;
+}
+
+.main-menu {
+  flex-grow: 1;
+}
+
+.main-menu ul {
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  padding-right: 50px;
+  justify-content: flex-end;
+}
+
+.main-menu ul li {
+  margin: 0 20px;
+  position: relative;
+}
+
+.main-menu ul li a {
+  text-decoration: none;
+  color: #000;
+  font-weight: bold;
+  height: 54px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.highlight {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: ' ';
+  position: absolute;
+  left: 0;
+  width: 100%;
+  height: 8px;
+  background-color: #00acdc;
+  bottom: -18.5px;
```

```suggestion
  bottom: 0;
```
links shuld have height on full header height. so bottom shouldn't be hardcoded and should be 0.